config/initializers/money.rb

```diff
@@ -0,0 +1,92 @@
+# encoding : utf-8
+
+MoneyRails.configure do |config|
+
+  # To set the default currency
+  #
+  # config.default_currency = :usd
+
+  # Set default bank object
+  #
+  # Example:
+  # config.default_bank = EuCentralBank.new
+
+  # Add exchange rates to current money bank object.
+  # (The conversion rate refers to one direction only)
+  #
+  # Example:
+  # config.add_rate "USD", "CAD", 1.24515
+  # config.add_rate "CAD", "USD", 0.803115
+
+  # To handle the inclusion of validations for monetized fields
+  # The default value is true
+  #
+  # config.include_validations = true
+
+  # Default ActiveRecord migration configuration values for columns:
+  #
+  config.amount_column = { prefix:      '', # column name prefix
+                           postfix:     '_cents', # column name  postfix
+                           column_name: nil, # full column name (overrides prefix, postfix and accessor name)
+                           type:        :bigint, # column type
+                           present:     true, # column will be created
+                           null:        false, # other options will be treated as column options
+                           default:     0
+  }
```

Comment:
Is this only necessary for migration purposes or does the runtime of MoneyRails need this as well?